quodlibet/quodlibet/ext/playorder/skip_zeros.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Nick Boultbee
+#           2017 Jason Heard
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+
+from gi.repository import Gtk
+
+from quodlibet import _, print_d, qltk
+from quodlibet.order import OrderInOrder
+from quodlibet.plugins import PluginConfigMixin
+from quodlibet.plugins.playorder import ShufflePlugin
+from quodlibet.qltk import Icons
+from quodlibet.qltk.ccb import ConfigCheckButton
+
+
+class SkipZeros(ShufflePlugin, OrderInOrder, PluginConfigMixin):
+    PLUGIN_ID = "skip_songs"
+    PLUGIN_NAME = _("Skip Songs")
+    PLUGIN_ICON = Icons.GO_JUMP
+    PLUGIN_DESC = _("Playback skips over songs with a rating of zero or with "
+                    "a non-empty 'skip' tag unless they are explicitly "
+                    "played.")
+
+    _CFG_SKIP_BY_RATING = 'skip_by_rating'
+    _CFG_SKIP_BY_TAG = 'skip_by_tag'
+
+    @classmethod
+    def PluginPreferences(self, window):
+        vb = Gtk.VBox(spacing=10)
+        vb.set_border_width(0)
+
+        # Matching Option
+        toggles = [
+            (self._CFG_SKIP_BY_RATING, _("Skip Songs with _Rating of Zero")),
+            (self._CFG_SKIP_BY_TAG,
+             _("Skip Songs with a Non-Empty 'skip' _Tag")),
+        ]
+        vb2 = Gtk.VBox(spacing=6)
```

Comment:
Do I need `vb2` along with `vb` here, or is that redundant?  I copied this code from the duplicate songs plugin, but don't have the textbox above the checkboxes.